File: java/h00.override/Shape.java
public class Shape {
   /******* FIELDS *******/
   protected Double width;
   protected Double height;

   /******* CONSTRUCTORS *******/
   public Shape(Double width, Double height)  {
     this.width = width;
     this.height = height;
   }

   public Shape(Integer width, Integer height)  {
     this.width  = new Double(width);   // convert to Double
     this.height = new Double(height);  // convert to Double
   }

   /******* METHODS *******/
   public Double calculateArea() {
      return 0.0;
   }
}

File: python/h00.override/Shape.py
#!/usr/bin/python
# class declaration
class Shape(object):
  ###### Initializer ######
  def __init__(self, width, height):
    self.width  = width / 1.0
    self.height = height / 1.0

  ###### Accessors ######
  def calculateArea(self):
    return 0.0
